Cargo.toml

```diff
@@ -42,15 +42,15 @@ rustc_version = "0.4"
 serde = "1.0.154"
 serde_json = "1.0.114"
 tracing = "0.1"
-zenoh = { version = "0.11.0-dev", git = "https://github.com/eclipse-zenoh/zenoh.git", branch = "dev/1.0.0", features = [
+zenoh = { version = "0.11.0-dev", git = "https://github.com/eclipse-zenoh/zenoh.git", branch = "dev/admin_keys_remapping", features = [
   "internal", "unstable", "plugins"
 ] }
-zenoh-ext = { version = "0.11.0-dev", git = "https://github.com/eclipse-zenoh/zenoh.git", branch = "dev/1.0.0", features = [
+zenoh-ext = { version = "0.11.0-dev", git = "https://github.com/eclipse-zenoh/zenoh.git", branch = "dev/admin_keys_remapping", features = [
   "unstable",
 ] }
-zenoh-plugin-rest = { version = "0.11.0-dev", git = "https://github.com/eclipse-zenoh/zenoh.git", branch = "dev/1.0.0", default-features = false }
+zenoh-plugin-rest = { version = "0.11.0-dev", git = "https://github.com/eclipse-zenoh/zenoh.git", branch = "dev/admin_keys_remapping", default-features = false }
 zenoh-plugin-ros2dds = { version = "0.11.0-dev", path = "zenoh-plugin-ros2dds/", default-features = false }
-zenoh-plugin-trait = { version = "0.11.0-dev", git = "https://github.com/eclipse-zenoh/zenoh.git", branch = "dev/1.0.0", default-features = false }
+zenoh-plugin-trait = { version = "0.11.0-dev", git = "https://github.com/eclipse-zenoh/zenoh.git", branch = "dev/admin_keys_remapping", default-features = false }
```

Comment:
To be updated after the merge of: https://github.com/eclipse-zenoh/zenoh/pull/1209